assets/scss/style.scss

```diff
@@ -0,0 +1,13 @@
+
+.card {
```

Comment:
Please modified in https://github.com/404busters/hkoscon-2017/blob/master/assets/scss/elements/_card.scss